content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -117,18 +153,22 @@ they will not be evicted when there are node problems such as a network partitio
 `TaintBasedEvictions` feature is not enabled, they are also not evicted in these scenarios, but
 due to hard-coded behavior of the NodeController rather than due to tolerations).
 
- They also tolerate following `NoSchedule` taints:
+They also tolerate following `NoSchedule` taints:
 
- - `node.kubernetes.io/memory-pressure`
  - `node.kubernetes.io/disk-pressure`
+ - `node.kubernetes.io/memory-pressure`
+ - `node.kubernetes.io/unschedulable` (after version 1.10)
 
 When the support to critical pods is enabled and the pods in a DaemonSet are
 labeled as critical, the Daemon pods are created with an additional
 `NoSchedule` toleration for the `node.kubernetes.io/out-of-disk` taint.
 
-Note that all above `NoSchedule` taints above are created only in version 1.8 or later if the alpha feature `TaintNodesByCondition` is enabled.
+Note that all above `NoSchedule` taints are created only in version 1.8 or later
+ if the alpha feature `TaintNodesByCondition` is enabled.
```

Comment:
This "Taints and Tolerations" section is becoming difficult to follow. 

Would you make it easier to read, such as using tables and clarify what is required when which feature is enabled?